configserver/src/main/java/com/yahoo/vespa/config/server/tenant/RotationsCache.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.config.server.tenant;
+
+import com.yahoo.config.provision.ApplicationId;
+import com.yahoo.path.Path;
+import com.yahoo.slime.ObjectTraverser;
+import com.yahoo.slime.Slime;
+import com.yahoo.vespa.applicationmodel.ClusterId;
+import com.yahoo.vespa.config.SlimeUtils;
+import com.yahoo.vespa.curator.Curator;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * @author ogronnesby
+ */
+public class RotationsCache {
+    private final Path cachePath;
+    private final Curator curator;
+
+    public RotationsCache(Path tenantPath, Curator curator) {
+        this.cachePath = tenantPath.append("rotationsCache-v2/");
+        this.curator = curator;
+    }
+
+    public Map<ClusterId, RotationAssignment> getRotationAssignment(ApplicationId applicationId) {
```

Comment:
These methods either read or write, consider calling them that so it's obvious to the caller that they perform IO. The type is already in the signature and doesn't have to be in the name.